pkg/compiler/codegen.go

```diff
@@ -583,12 +584,16 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 						} else {
 							c.scope.newLocal(id.Name)
 						}
-						c.registerDebugVariable(id.Name, t.Type)
+						if !multiRet {
+							c.registerDebugVariable(id.Name, t.Type)
```

Comment:
@fyrchik, I'm not sure whether we need to add such variables to debug info. For some reason we don't add it for assignments, not sure, why: https://github.com/nspcc-dev/neo-go/blob/f8857c5ebe89eac226c718d603613e577101aa68/pkg/compiler/codegen.go#L614-L618